src/nvme/mi.h

```diff
@@ -367,6 +367,34 @@ struct nvme_mi_admin_resp_hdr {
 	__le32	cdw0, cdw1, cdw3;
 } __attribute__((packed));
 
+/**
+ * struct nvme_mi_service_req_hdr - servicing command request header.
+ * @hdr: Generic MI message header
+ * @opcode: servicing command opcode (using &enum nvme_service_opcode)
+ * @tag: flag - be used to sent and return
+ * @cpsp: Control Primitive Specific Parameter
+ */
+struct nvme_mi_service_req_hdr {
+	struct nvme_mi_msg_hdr hdr;
+	__u8	opcode;
+	__u8	tag;
+	__le16	cpsp;
+} __attribute((packed));
```

Comment:
This isn't just the header, right? It's the structure of the entire control primitive message. In which case, we should name it accordingly - maybe just remove the `_hdr` suffix.

Reply:
done